webtau-report/src/main/java/com/twosigma/webtau/report/ReportTestEntry.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2018 TWO SIGMA OPEN SOURCE, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twosigma.webtau.report;
+
+import com.twosigma.webtau.reporter.TestResultPayload;
+import com.twosigma.webtau.reporter.TestStatus;
+import com.twosigma.webtau.reporter.TestStep;
+import com.twosigma.webtau.reporter.stacktrace.StackTraceCodeEntry;
+import com.twosigma.webtau.reporter.stacktrace.StackTraceUtils;
+import com.twosigma.webtau.utils.FileUtils;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static com.twosigma.webtau.cfg.WebTauConfig.getCfg;
+import static com.twosigma.webtau.reporter.TestStatus.*;
+
+public class ReportTestEntry {
```

Comment:
If only Java had data classes 😄

Reply:
Is this Kotlin plug?

Reply:
I guess but I'm not sure it makes sense to introduce a new language just to help with one POJO!